src/components/CreateAccount.js

```diff
@@ -200,19 +207,19 @@ class CreateAccount extends Component {
         }
 
         let errorMsg, usernameWarning, emailWarning, passwordWarning;
-        let buttonClass = 'btn btn-primary';
-        let usernameClass = 'form-group has-';
-        let emailClass = 'form-group has-';
-        let passwordClass = 'form-group has-';
+        let usernameClass = 'has-';
+        let emailClass = 'has-';
+        let passwordClass = 'has-';
```

Comment:
lol. just do `let usernameClass, emailClass, passwordClass`;`

Then in the valid / invalid stuff, do `let usernameClass = 'has-error'` so you can avoid this weird code style lol